[WIP]add document: how to migrate data to tidb (#1271)
pingcap/docs: v3.0/how-to/migrate/to-tidb.md

---
title: Migrate Data from TiDB to TiDB/MySQL
summary: migrate data from master TiDB clister to slave TiDB cluster or MySQL.
category: how-to

> aliases: ['/docs/op-guide/migration/']

Reviewer: ```suggestion

```
No need to provide aliases for a totally new doc.

---

# Migrate Data from TiDB to TiDB/MySQL

## Pre-requirements

> * you must already [deploy pump cluster](https://pingcap.com/docs/v3.0/reference/tools/tidb-binlog/deploy/), don't need to deploy drainer.

Reviewer: ```suggestion
* you must already [deploy pump cluster](/reference/tools/tidb-binlog/deploy/), no need to deploy drainer.
```

## Temporal logic of migration opeations

1.  Obtain `initial_commit_ts`. 

> 2. Use `mydumper` to export data from master TiDB cluster

Reviewer: ```suggestion
2. Use [mydumper](/reference/tools/mydumper/) to export data from master TiDB cluster
```
Link to the mydumper instructions doc.

> 3. Use `loader` to import the data into slave TiDB cluster or MySQL.

Reviewer: ```suggestion
3. Use [loader](/reference/tools/loader.md) to import the data into slave TiDB cluster or MySQL.
```
